components/post_view/post_header/post_header.tsx

```diff
@@ -186,6 +186,9 @@ export default class PostHeader extends React.PureComponent<Props> {
                         showTimeWithoutHover={this.props.showTimeWithoutHover}
                         hover={this.props.hover}
                         isLastPost={this.props.isLastPost}
+                        isReadOnly={null}
```

Comment:
None of these changes should be necessary, especially the `actions` portion. Again, `actions` is provided by the redux connector and shouldn't cause a type error if done correctly. Sounds like something else is wrong here.

Reply:
Removing these 3 lines causes the `check-types` to fail.

```
components/post_view/post_header/post_header.tsx:178:22 - error TS2739: Type '{ post: Post; handleCommentClick: (event: MouseEvent<Element, MouseEvent>) => void; handleCardClick: (post: Post) => void; handleDropdownOpened: (opened: boolean) => void; ... 5 more ...; isLastPost: boolean | undefined; }' is missing the following properties from type 'Props': isReadOnly, actions, shouldShowDotMenu

178                     <PostInfo
```

Reply:
I'm not sure how to solve this @devinbinnie

Reply:
@devinbinnie - this is now fixed.